packages/realm-react/package.json

```
@@ -13,7 +13,8 @@
   },
   "dependencies": {
     "lodash": "^4.17.21",
-    "realm": ">=10"
+    "realm": ">=10",
+    "@realm.io/common": "*"
```

I suspect existing package locks won't upgrade this if the version isn't bumped.
As we have [`forceLocal` enabled](https://github.com/realm/realm-js/blob/master/lerna.json#L12), there shouldn't be an issue with this version not matching the package.json of `react-common`.

```suggestion
    "@realm.io/common": "^0.1.2"
```

---

Got it, thanks, fixed